src/main/perf/LineFileDocs.java

```diff
@@ -471,6 +486,29 @@ static Document cloneDoc(Document doc1) {
     return doc2;
   }
 
+  private String getRandomWord(String body) {
```

Comment:
I would prefer, if possible, that we move this random word selection to the building of the line docs file instead, i.e. add it as a new column on our LFD format.  We try to keep the indexing benchmark pure indexing -- just suck lines out of the file, convert to Lucene `Document` as efficiently as possible.  This way we do our best to ensure that the indexing throughput benchmark really is measuring Lucene's indexing performance and not also the cost of building documents (even though the latter will always have to be non-zero).

Reply:
However, we unfortunately have lost the ability to re-generate a new `LineFileDocs` (LFD) file from original sources (Wikipedia export).  There is an [issue open](https://github.com/mikemccand/luceneutil/issues/91) and [far-along PR](https://github.com/mikemccand/luceneutil/issues/91) as well, but [a WTF (wow that's funny) emerged and stopped me cold](https://github.com/mikemccand/luceneutil/issues/91#issuecomment-740787842).  So, perhaps we could make a silly little tool that does this same random word extraction, takes in an existing LFD, and appends a new column to it, and we switch default benchmarks to use this new LFD.

Reply:
I wrote a quick script to read the `enwiki-20120502-lines-1k.txt` write to a new `enwiki-20120502-lines-1k-with-random-label.txt`. It would need to be run after the setup script is run and the original enwiki file is extracted. Do you think it's maybe worth it to call this in the setup script as well? I would need to figure out how to read a `.lzma` in Python file, but it should be doable.